target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/rtl931x.c

```diff
@@ -468,7 +468,8 @@ void rtl931x_print_matrix(void)
 
 	for (int i = 0; i < 64; i++) {
 		rtl_table_read(r, i);
-		pr_info("> %08x %08x\n", sw_r32(rtl_table_data(r, 0)), sw_r32(rtl_table_data(r, 1)));
+		pr_info("> %08x %08x\n", sw_r32(rtl_table_data(r, 0)),
+			sw_r32(rtl_table_data(r, 1)));
```

Comment:
This is touched in two commits of this PR. Please drop the matrix modification from first commit ("realtek: rtl931x: Fix traffic on upper ports") and change it only once here to the final version.